src/components/layers-panel/arcgis-import-panel/arcgis-import-panel.tsx

```diff
@@ -196,13 +190,91 @@ export const ArcGisImportPanel = ({ onImport, onCancel }: InsertLayerProps) => {
     }
   };
 
-  const formatDate = (date: number) => {
-    const formatter = new Intl.DateTimeFormat("en-US", {
-      month: "long",
-      day: "2-digit",
-      year: "numeric",
-    });
-    return formatter.format(date);
+  type Column = {
+    width: number;
+    fontWeight?: number;
+    dataColumnName?: ArcGisContentColumnName;
+    sortDataColumnName?: ArcGisContentColumnName;
+    columnName?: string;
+  };
+
+  const columns: Column[] = [
+    {
+      width: 44,
+    },
+    {
+      width: 343,
+      fontWeight: 700,
+      dataColumnName: "title",
+      columnName: "Title",
+    },
+    {
+      width: 149,
+      dataColumnName: "createdFormatted",
+      sortDataColumnName: "created",
+      columnName: "Date",
+    },
+  ];
+
+  const renderHeaderCell = (column: Column): JSX.Element => {
+    const sortDataColumnName =
+      column.sortDataColumnName || column.dataColumnName;
+    return (
+      <TableHeaderCell
+        width={column.width}
+        key={sortDataColumnName ? sortDataColumnName : ""}
```

Comment:
can we add an `id` field to the columns config to use it in JSX?

Reply:
Done.